Add new device Ubiquiti Litebeam AC Gen2 from openwrt staging/stintel.git and use the same for Litebeam AC Gen1 (LBE-AC5-23).

Additional changes:
* use "kmod-ath10k-ct-smallbuffers" instead of "kmod-ath10k-ct"

--- on_firmware.orig/openwrt/target/linux/ath79/base-files/etc/board.d/02_network
+++ on_firmware/openwrt/target/linux/ath79/base-files/etc/board.d/02_network
@@ -36,6 +36,8 @@ ath79_setup_interfaces()
 	ubnt,bullet-m|\
 	ubnt,bullet-m-xw|\
 	ubnt,lap-120|\
+	ubnt,litebeam-ac-gen1|\
+	ubnt,litebeam-ac-gen2|\
 	ubnt,nanobeam-ac|\
 	ubnt,nanostation-ac-loco|\
 	ubnt,rocket-m|\

--- on_firmware.orig/openwrt/target/linux/ath79/base-files/etc/hotplug.d/firmware/11-ath10k-caldata
+++ on_firmware/openwrt/target/linux/ath79/base-files/etc/hotplug.d/firmware/11-ath10k-caldata
@@ -170,6 +170,8 @@ case "$FIRMWARE" in
 	ubnt,unifiac-mesh|\
 	ubnt,unifiac-mesh-pro|\
 	ubnt,lap-120|\
+	ubnt,litebeam-ac-gen1|\
+	ubnt,litebeam-ac-gen2|\
 	ubnt,nanobeam-ac|\
 	ubnt,nanostation-ac|\
 	ubnt,nanostation-ac-loco|\

--- on_firmware.orig/openwrt/target/linux/ath79/image/generic-ubnt.mk
+++ on_firmware/openwrt/target/linux/ath79/image/generic-ubnt.mk
@@ -138,6 +138,24 @@ define Device/ubnt_nanobeam-ac
 endef
 TARGET_DEVICES += ubnt_nanobeam-ac
 
+define Device/ubnt_litebeam-ac-gen1
+  $(Device/ubnt-wa)
+  DEVICE_TITLE := Ubiquiti Litebeam AC Gen1
+  DEVICE_PACKAGES += kmod-ath10k-ct-smallbuffers ath10k-firmware-qca988x-ct
+  IMAGE_SIZE := 15744k
+  IMAGE/factory.bin := $$(IMAGE/sysupgrade.bin) | mkubntimage-split
+endef
+TARGET_DEVICES += ubnt_litebeam-ac-gen1
+
+define Device/ubnt_litebeam-ac-gen2
+  $(Device/ubnt-wa)
+  DEVICE_TITLE := Ubiquiti Litebeam AC Gen2
+  DEVICE_PACKAGES += kmod-ath10k-ct-smallbuffers ath10k-firmware-qca988x-ct
+  IMAGE_SIZE := 15744k
+  IMAGE/factory.bin := $$(IMAGE/sysupgrade.bin) | mkubntimage-split
+endef
+TARGET_DEVICES += ubnt_litebeam-ac-gen2
+
 define Device/ubnt_nanostation-ac
   $(Device/ubnt-wa)
   DEVICE_TITLE := Ubiquiti Nanostation AC

--- /dev/null
+++ on_firmware/openwrt/target/linux/ath79/dts/ar9342_ubnt_litebeam-ac-gen1.dts
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: GPL-2.0
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9342_ubnt_wa.dtsi"
+
+/ {
+	compatible = "ubnt,litebeam-ac-gen1", "ubnt,wa", "qca,ar9342";
+	model = "Ubiquiti LiteBeam AC Gen1";
+};
+
+&mdio0 {
+	status = "okay";
+
+	phy-mask = <4>;
+	phy4: ethernet-phy@4 {
+		phy-mode = "rgmii";
+		reg = <4>;
+	};
+};
+
+&eth0 {
+	status = "okay";
+
+	/* default for ar934x, except for 1000M and 10M */
+	pll-data = <0x02000000 0x00000101 0x00001313>;
+
+	mtd-mac-address = <&eeprom 0x0>;
+
+	phy-mode = "rgmii";
+	phy-handle = <&phy4>;
+
+	gmac-config {
+		device = <&gmac>;
+		rxd-delay = <3>;
+		rxdv-delay = <3>;
+	};
+};

--- /dev/null
+++ on_firmware/openwrt/target/linux/ath79/dts/ar9342_ubnt_litebeam-ac-gen2.dts
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: GPL-2.0
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9342_ubnt_wa.dtsi"
+
+/ {
+	compatible = "ubnt,litebeam-ac-gen2", "ubnt,wa", "qca,ar9342";
+	model = "Ubiquiti LiteBeam AC Gen2";
+};
+
+&mdio0 {
+	status = "okay";
+
+	phy-mask = <4>;
+	phy4: ethernet-phy@4 {
+		phy-mode = "rgmii";
+		reg = <4>;
+	};
+};
+
+&eth0 {
+	status = "okay";
+
+	/* default for ar934x, except for 1000M and 10M */
+	pll-data = <0x02000000 0x00000101 0x00001313>;
+
+	mtd-mac-address = <&eeprom 0x0>;
+
+	phy-mode = "rgmii";
+	phy-handle = <&phy4>;
+
+	gmac-config {
+		device = <&gmac>;
+		rxd-delay = <3>;
+		rxdv-delay = <3>;
+	};
+};
